src/js/utils/propTypes.js

```diff
@@ -0,0 +1,6 @@
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line import/prefer-default-export
+export const FormErrorPropType = PropTypes.shape({
+  message: PropTypes.string,
+});
```

Comment:
so if this file will be used for defining our custom propTypes, you can use default export like that:
```suggestion
import PropTypes from 'prop-types';

const FormErrorPropType = PropTypes.shape({
  message: PropTypes.string,
});

export default propTypes = {
   // Add here newly created custom propTypes
   FormErrorPropType,
};
```

Reply:
this was intentional from my side.
The above example with default export will not get optimized by webpack treeshaking algorithm which should omit the unused code from being included in the bundle.
So in such cases, it is best to export it individually.